sklearn/cluster/spectral.py

```diff
@@ -362,6 +367,12 @@ class SpectralClustering(BaseEstimator, ClusterMixin):
         The number of parallel jobs to run.
         If ``-1``, then the number of jobs is set to the number of CPU cores.
 
+    copy : bool, optional, default: True
+        If copy is False, then the laplacian matrix is copied in the affinity
```

Comment:
"is copied" here is confusing in the copy is False case. In general this is not very clear to me.